posthog/management/commands/update_batch_export_schedules.py

```diff
@@ -11,29 +15,61 @@ class Command(BaseCommand):
+
+        try:
+            sync_batch_export(batch_export, created=False)
+        except Exception:
+            logger.exception("Error updating batch export schedule", batch_export_id=str(batch_export.id))
 
     def handle(self, **options):
+        logger.setLevel(logging.INFO)
         batch_export_id = options["batch_export_id"]
-        # verbose = options["verbosity"] > 1
 
-        if not batch_export_id:
-            raise CommandError("Batch export ID is required")
+        batch_exports: list[BatchExport] = []
 
-        # TODO - run in a loop once we support multiple batch exports
-        try:
-            batch_export: BatchExport = BatchExport.objects.get(id=batch_export_id, deleted=False)
-        except BatchExport.DoesNotExist:
-            raise CommandError("Batch export not found")
-        except BatchExport.MultipleObjectsReturned:
-            raise CommandError("More than one existing batch export found (this should never happen)!")
+        if batch_export_id:
+            try:
+                batch_export: BatchExport = BatchExport.objects.get(id=batch_export_id, deleted=False)
+                batch_exports.append(batch_export)
+            except BatchExport.DoesNotExist:
+                raise CommandError("Batch export not found")
+            except BatchExport.MultipleObjectsReturned:
+                raise CommandError("More than one existing batch export found (this should never happen)!")
+        elif options["destination_type"] or options["team_id"]:
+            query_set = BatchExport.objects.filter(deleted=False)
 
-        logger.info("Updating batch export schedule...", batch_export_id=batch_export.id)
+            if options["destination_type"]:
+                query_set = query_set.filter(destination__type=options["destination_type"])
 
-        try:
-            sync_batch_export(batch_export, created=False)
-        except Exception:
-            logger.exception("Error updating batch export schedule", batch_export_id=batch_export.id)
+            if options["team_id"]:
+                query_set = query_set.filter(team__id=int(options["team_id"]))
+
+            batch_exports = list(query_set)
+        else:
+            # Do we want to allow it to run on all batch exports?
+            raise CommandError("No batch export ID or destination type or team ID provided")
+
+        if len(batch_exports) == 0:
+            raise CommandError("No batch exports found")
+
+        if not settings.TEST:
+            confirm = input(f"\n\tWill update schedules for {len(batch_exports)} batch exports. Proceed? (y/n) ")
+            if confirm.strip().lower() != "y":
+                logger.info("Aborting")
+                return
+
+        for batch_export in batch_exports:
+            self._update_batch_export_schedule(batch_export)
+            if not settings.TEST:
+                # add a small delay to avoid flooding Temporal with requests
+                time.sleep(0.5)
```

Comment:
I think you can just send it? Not sure why we are protecting temporal

Reply:
ok, fair enough, will remove it and then can always add it back in if we're seeing issues 👍